src/client/compass.ts

```diff
@@ -39,7 +39,7 @@ const throwIfErrors = function throwIfSdkErrors(method: string, errors: SdkError
 
 export const createComponent = async (cloudId: string, project: ImportableProject): Promise<Component | never> => {
   const { name, description, typeId, labels, url, ownerId } = project;
-  const formattedLabels = formatLabels(labels);
+  const formattedLabels = formatLabels(labels).slice(0, 19);
```

Comment:
Could be a good idea to extract this number of max labels to a constant. If exported, you could also re-use it in all parts where it would be meaningful, for consistency.

Reply:
good idea, updated!